src/query/service/src/pipelines/executor/executor_graph.rs

```diff
@@ -241,8 +241,15 @@ impl ExecutingGraph {
                 if state_guard_cache.is_none() {
                     state_guard_cache = Some(node.state.lock().unwrap());
                 }
-
-                let processor_state = match node.processor.event()? {
+                let event = node.processor.event()?;
+                tracing::debug!(
```

Comment:
we should use trace level here.

Reply:
fixed in https://github.com/datafuselabs/databend/pull/7927